packages/salesforcedx-vscode-apex/src/index.ts

```diff
@@ -65,6 +67,8 @@ export async function activate(context: vscode.ExtensionContext) {
     isLanguageClientReady,
     getApexTests
   };
+
+  telemetryService.sendExtensionActivationEvent(extensionHRStart);
```

Comment:
This times are close (not exact) as the ones shown by running `Developer: Startup Performance` in vscode.